### PR TITLE
[Azure Analysis Services] Add new version of API spec and update backup configuration

### PR DESCRIPTION
**Pull request by taiwu**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by vishrutshah** [comment]
@sarangan12 Based on my previous review of https://github.com/Azure/azure-rest-api-specs/pull/1287#discussion_r121547676, I am taking look into this so i'll assign it to me . If other reviews comes up on my plate, I'll give you if you have time :)

File: arm-analysisservices/2017-06-09/swagger/analysisservices.json [hunk 1]
```diff
@@ -519,8 +519,9 @@
         "asAdministrators": {
           "$ref": "#/definitions/ServerAdministrators"
         },
-        "backupConfiguration": {
-          "$ref": "#/definitions/BackupConfiguration"
+        "BackupBlobContainerUri": {
+          "type": "string",
+		  "description": "The container URI of backup blob."
```
Comment (vishrutshah) [inline arm-analysisservices/2017-06-09/swagger/analysisservices.json]: minor: could you please format  json with 2 spaces as the all other things are formatted as 2 spaces in this swagger.

File: arm-analysisservices/2017-06-09/swagger/analysisservices.json [hunk 2]
```diff
@@ -519,8 +519,9 @@
         "asAdministrators": {
           "$ref": "#/definitions/ServerAdministrators"
         },
-        "backupConfiguration": {
-          "$ref": "#/definitions/BackupConfiguration"
+        "BackupBlobContainerUri": {
```
Comment (vishrutshah) [inline arm-analysisservices/2017-06-09/swagger/analysisservices.json]: Why is this not camelCased? 

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/241267885#L1273

**Review by vishrutshah — CHANGES_REQUESTED** [review]
Hi @taiwu,

Thanks for sending the new review with the 2 commits to help us with the review. Being this a new API version, I'd like to ask following questions:

- [ ] Is this service live / deployed?
- [ ] Do you have `/operations` api modeled/exposed?
- [ ] Please review the linter errors https://travis-ci.org/Azure/azure-rest-api-specs/jobs/241267885#L1162
- [ ] We'd like to request [x-ms-examples](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md#why-x-ms-examples) to help us with the model validation. 

Please let us know your thoughts. Thanks!

**Comment by vishrutshah** [comment]
Thanks @taiwu for fixing the indentation and camelCase. I'd appreciate if you can comment on each of the general questions that we have asked. That's help me understand and get more insights into the status of the service. Thanks!

**Comment by salameer** [comment]
Hi @taiwu

Can you please address the comment soon :)

**Comment by taiwu** [comment]
Is this service live / deployed? yes
Do you have /operations api modeled/exposed? yes
For linter errors and warnings, still working on them.

**Comment by vishrutshah** [comment]
@taiwu Thanks for the answers, Let us know once you have worked through the linter issues. Thanks!

**Comment by taiwu** [comment]
After discussions with our team, we decide to close this pull request. What we will change is keep the current version instead of creating new version, and add new mutable property for our backupBlobContainerUri. Sorry for the inconvenience. 
The original planned to remove property is actually not working hence deprecated, it is no harm to delete them.

**Comment by taiwu** [comment]
I will create a new pull request for the change.

**Comment by vishrutshah** [comment]
Closing this PR in favor of https://github.com/Azure/azure-rest-api-specs/pull/1326

**Pull request closed without merging**